backend/config.ts

```diff
@@ -38,11 +38,7 @@ export const app = {
     scheduleRedirectForPrisons: process.env.HOMEPAGE_SCHEDULE_REDIRECT_FOR_PRISONS || '',
     exemptions: process.env.HOMEPAGE_REDIRECT_EXEMPTIONS || '',
   },
-  bvlsMasteredVlpmFeatureToggleEnabled: process.env.BVLS_MASTERED_VLPM_FEATURE_TOGGLE_ENABLED === 'true',
-  bvlsMasteredAppointmentTypes: [
-    'VLB',
-    process.env.BVLS_MASTERED_VLPM_FEATURE_TOGGLE_ENABLED === 'true' ? 'VLPM' : undefined,
-  ].filter(Boolean),
+  bvlsMasteredAppointmentTypes: ['VLB', 'VLPM'].filter(Boolean),
   gitRef: process.env.GIT_REF || '',
```

Comment:
Does it need the filter(Boolean) any more?

Reply:
This isn't the toggle.  This is the supported appointment types.

Reply:
Yup, I realise that. I see what its doing on 2nd look - the `.filter(Boolean)` part is there so it can be used in a certain way. All good.